docs/source/user_guide/box_model/index.rst

```diff
@@ -1,13 +1,61 @@
 Box model
 =========
+This section details the components of the box model, including reactions, mechanisms, conditions, and solutions. To use the MusicBox model, import::
+    
+    from acom_music_box import MusicBox, Conditions
 
-This section details the components of the box model, including reactions, mechanisms, conditions, and solutions.
+Using the previously defined in-code mechanism (see :ref:`Defining chemical species <species>`), a box model can now be created and initialized with it::
+    
+    box_model = MusicBox()
+    box_model.load_mechanism(mechanism)
 
-.. toctree::
-   :maxdepth: 1
+Initial and evolving conditions
```

Comment:
For consistency purpose, suggest changing to `2. setting initial and evolving conditions`